nnef/src/ast.rs

```diff
@@ -8,8 +8,9 @@ pub mod quant;
 #[derive(Clone, Debug)]
 pub struct ProtoModel {
     pub doc: Document,
-    pub tensors: Vec<(String, Arc<Tensor>)>,
+    pub tensors: HashMap<String, Arc<Tensor>>,
```

Comment:
I hope this is not going to bite us someplace it hurts. The reason that was a Vec was (I think) to have a reproducible ordering or tensor. We'll see.

Reply:
Yeah, I guess we will if we have to. I'm just not sure why I picked a Vec instead of a HashMap in the first place... Was it for some unit tests ? Everything seems to be green right now, so I'm not going to worry about it for now.